index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portafolio</title>
+    <link rel="stylesheet" href="./css/styles.css">
+</head>
+<body>
+        <header class="nav-menu">
+            <h2 class="mylogo">My Logo</h2>
+           <nav>
+               <div class="menu-header">
+                   <img id="icon-menu" src="/css/Icon-Menu.png" alt="menu">
+               </div>
+        </nav>
+        </header>
+        <main class="imagen_bg">
+            <section>
+                <div class="section1">
+                 <h1 class="presentation">I'm Andrea <br> Glad to see <br> you!</h1>
+            <p class="description">I'm a software developer! I can help you build a product, 
+            feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.</p>
+            <h3 class="connect" >LET'S CONNECT</h3> 
```

Comment:
- [ ] Let's make the text `LET'S CONNECT` clickable by using a `button` or  an `a` tag.